pyproject.toml

```diff
@@ -21,14 +21,14 @@ classifiers = [
 
 dependencies = [
   "toml>=0.10.2",
-  "requests",
+  "requests>=2.32.0,<3.0.0",
```

Comment:
Since mxpy is an end-application, we can also have precise / fixed versions (for all, actually).

Reply:
will do this in future PRs.